control-plane/agents/src/bin/core/tests/controller/mod.rs

```diff
@@ -290,7 +291,7 @@ async fn etcd_pagination() {
 
     cluster.restart_core().await;
     cluster
-        .volume_service_liveness(None)
+        .volume_service_liveness(Some(TimeoutOptions::new().with_max_retries(10)))
```

Comment:
you need to rebase, I already fixed this I think

Reply:
Ahh, I see the fix is different from what I did, I missed this, will remove.

Reply:
Done.